frontend/purposeful-frontend/app/page.js

```diff
@@ -22,200 +22,59 @@ import { RxPlus } from "react-icons/rx";
-                      <FormControl id="domain">
-                        <FormLabel>Domains</FormLabel>
-                        <HStack placeholder="First name">{domains_sel}</HStack>
-                      </FormControl>
-                    )}
-                  </Field>
-                  {domainContainer}
-                </Box>
-                <Box width={"100%"} display={"flex"}>
-                  <Field name="Topic">
-                    {({ field, form }) => (
-                      <FormControl
-                        id="topic"
-                        isInvalid={form.errors.topic && form.touched.topic}>
-                        <FormLabel>Topics</FormLabel>
-                        <HStack>{topics_sel}</HStack>
-                      </FormControl>
-                    )}
-                  </Field>
-                  {topicContainer}
-                </Box>
-                <Box width={"100%"} display={"flex"}>
-                  <Field name="Tech">
-                    {({ field, form }) => (
-                      <FormControl
-                        id="tech"
-                        isInvalid={form.errors.tech && form.touched.tech}>
-                        <FormLabel>Technologies</FormLabel>
-                        <HStack>{techs_sel}</HStack>
-                      </FormControl>
-                    )}
-                  </Field>
-                  {techContainer}
-                </Box>
-                {/* <Box width={"100%"} display={"flex"}>
-                  <Button
-                    width={"100%"}
-                    bg={"blue.400"}
-                    color={"white"}
-                    _hover={{
-                      bg: "blue.500",
-                    }}
-                    type="submit">
-                    Search
-                  </Button>
-                </Box> */}
-              </HStack>
-            </Form>
-          )}
-        </Formik>
+        position="fixed"
+        // width={"20%"}
+        width={"20%"}
+        display={"flex"}
+        className={styles.ideaFilter}>
+        {/* This is the components to update eventually. */}
+        {/* The components updates the list of ideas. */}
+        {/* <IdeaFilter setIdeas={setIdeas} /> */}
+        <IdeaFilter2 setIdeas={setIdeas} />
       </Box>
-      <List minWidth="100%">
-        {ideas.map((idea) => (
-          <ListItem>
-            <IdeaCard key={idea} />
-          </ListItem>
-        ))}
-      </List>
-    </Stack>
+      <Stack className={styles.ideas}>
+        <List>
+          {ideas.map((idea, index) => (
+            <ListItem
+              key={index}
+              width={"100%"}
+              display={"flex"}
+              justifyContent={"center"}
+              margin={"50px 0px"}>
+              <Box className={styles.ideaCard}>
+                {/* Map the ideas into individual cards */}
+                {/* Remove the hardcoded objects once real ideas are in the list */}
```

Comment:
Could you clean up the comments and remove the ones that are no longer relevant?